packages/manager/apps/web/client/app/hosting/dashboard/translations/Messages_fr_FR.json

```diff
@@ -381,10 +381,12 @@
   "hosting_tab_databases_dumps_error_fetch": "Une erreur est survenue lors de la récupération de vos sauvegardes.",
   "hosting_tab_DATABASES_configuration_copy_title": "Sélectionnez la base de données de destination",
   "hosting_tab_DATABASES_configuration_copy_action_button": "Copier la base de données",
+  "hosting_tab_DATABASES_configuration_information_message": "Que faire si aucune base de données de destination n’apparait ?",
```

Comment:
```suggestion
  "hosting_tab_DATABASES_configuration_information_message": "Que faire si aucune base de données de destination n'apparait ?",
```

Reply:
Oh dear ! I copy paste again and dit not see that thks!